src/symbols/analyzer.rs

```diff
@@ -1233,23 +1272,14 @@ impl<'a> SymbolFiller<'a> {
         continue; // don't fill the types on implementation signatures as they're private
       }
 
-      self.fill_module_item(
-        module_item,
-        &|name, symbol_id| {
-          self.builder.exports.insert(name, symbol_id);
-        },
-        &|symbol_id| {
-          self.builder.child_ids.borrow_mut().insert(symbol_id);
-        },
-      );
+      self.fill_module_item(module_item, module_symbol);
     }
   }
 
   fn fill_module_item(
     &self,
     module_item: &ModuleItem,
-    add_export: &impl Fn(String, SymbolId),
-    add_child: &impl Fn(SymbolId),
+    module_symbol: &SymbolMut,
```

Comment:
It's quite nice to now be able to pass these around without worrying about the borrow checker. The performance should be better after this change too because it's no longer doing so many hashmap lookups.